src/unstract/sdk/file_storage/helper.py

```diff
@@ -67,3 +67,24 @@ def local_file_system_init() -> AbstractFileSystem:
                 f" file system {e}"
             )
             raise FileStorageError(str(e)) from e
+
+
+def skip_local_cache(func):
+    def wrapper(*args, **kwargs):
+        try:
+            return func(*args, **kwargs)
+        except FileNotFoundError:
+            # FileNotFound could have been caused by stale cache.
+            # Hence invalidate cache and retry again
+            args[0].fs.invalidate_cache()
+            try:
```

Comment:
Nested try catcch not reccomented. Make it as some method.

Reply:
If this is not a blocker, can I address this in next  PR? Almost close to publishing rc3 to unblock OSS break... :(